iwyu_include_picker.cc

```diff
@@ -1414,14 +1416,13 @@ void IncludePicker::AddMappingsFromFile(const string& filename,
       return;
     }
 
-    for (MappingNode::iterator it = mapping->begin();
-        it != mapping->end(); ++it) {
+    for (KeyValueNode &node : *mapping) {
```

Comment:
I don't like shadowing `node` variable in outer loop. What if we name variable in outer loop `array_item_node` and this one `mapping_item_node`?